### PR TITLE
Chore: Complete LBP Admin Dashbaord

### PR DESCRIPTION
**Pull request by codebuster22**

### CODE REVIEW

File: src/admin/lbps/dashboard/dashboard.html [hunk 1]
```diff
@@ -28,38 +28,46 @@
...
+            <div class="featureContainer" if.bind="selectedLbp.uninitialized">
               <div class="featureTitle">
-                Fund LBP
+                Approve tokens and Fund LBP
+              </div>
+              <div class="featureDescription">
+                ${selectedLbp.projectTokenInfo.name} (${selectedLbp.projectTokenInfo.symbol}) <img class="tokenImage" src.to-view="selectedLbp.projectTokenInfo.logoURI"></img> required to fund the LBP: ${selectedLbp.startingProjectTokenAmount.toString() | ethwei:selectedLbp.projectTokenInfo.decimals}
+              </div>
+              <div class="featureDescription">
+                ${selectedLbp.fundingTokenInfo.name} (${selectedLbp.fundingTokenInfo.symbol}) <img class="tokenImage" src.to-view="selectedLbp.fundingTokenInfo.logoURI"></img> required to fund the LBP: ${selectedLbp.startingFundingTokenAmount.toString() | ethwei:selectedLbp.projectTokenInfo.decimals}
```
Comment (dkent600) [inline src/admin/lbps/dashboard/dashboard.html:39]: Should be using funding token decimals

File: src/admin/lbps/dashboard/dashboard.html [hunk 2]
```diff
@@ -28,38 +28,46 @@
             ${selectedLbp.metadata.general.projectName}
           </div>
           <div if.bind="selectedLbp.address">
-            <div class="featureContainer" if.bind="!selectedLbp.hasEnoughProjectTokens">
+            <div class="featureContainer" if.bind="selectedLbp.uninitialized">
               <div class="featureTitle">
-                Fund LBP
+                Approve tokens and Fund LBP
```
Comment (dkent600) [inline src/admin/lbps/dashboard/dashboard.html:33]: Can you please make this a little cleaner, like:

Required to fund the LBP:

   [XXX] [funding token info] 
   [YYY] [project token info]

**Review by dkent600 — CHANGES_REQUESTED** [review]
Please "connect" this PR to the ticket that motivates it, thanks.  Per our PR guidelines.

File: src/entities/LbpManager.ts [hunk 3]
```diff
@@ -333,12 +334,25 @@ export class LbpManager implements ILaunch {
       });
   }
 
-  public async setSwapEnabled(state: boolean): Promise<TransactionReceipt> {
+  public async setSwapDisabled(): Promise<TransactionReceipt> {
```
[the remark this answers is not in the document]
Comment (codebuster22) [reply src/entities/LbpManager.ts:340]: on it.

File: src/admin/lbps/dashboard/dashboard.html (hunk 1 above)
Comment (codebuster22) [reply src/admin/lbps/dashboard/dashboard.html:39]: ohh yes, doing it now

File: src/admin/lbps/dashboard/dashboard.html (hunk 2 above)
Comment (codebuster22) [reply src/admin/lbps/dashboard/dashboard.html:33]: okay, will do it.

File: src/admin/lbps/dashboard/dashboard.html [hunk 4]
```diff
@@ -28,38 +28,46 @@
...
-                ${selectedLbp.projectTokenInfo.name} (${selectedLbp.projectTokenInfo.symbol}) <img class="tokenImage" src.to-view="selectedLbp.projectTokenInfo.logoURI"></img> required to fund the LBP: ${selectedLbp.feeRemainder.add(selectedLbp.lbpRemainder) | ethwei:selectedLbp.projectTokenInfo.decimals}.
                 Note this will require your approval of three transactions.
               </div>
               <div class="button1 small" click.delegate="fund()">
-                Fund
+                Approve tokens and Fund
               </div>
             </div>
-            <div class="featureContainer">
+            <div class="featureContainer" if.bind="selectedLbp.poolFunded && selectedLbp.isDead && selectedLbp.poolTokenBalance != 0">
```
[the remark this answers is not in the document]
Comment (codebuster22) [reply src/admin/lbps/dashboard/dashboard.html:48]: ohh yes, doing it now,

[the remark this answers is not in the document]
Comment (codebuster22) [reply src/admin/lbps/dashboard/dashboard.html]: done

File: src/entities/LbpManager.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (codebuster22) [reply src/entities/LbpManager.ts]: done, it worked thanks.

File: src/admin/lbps/dashboard/dashboard.html (hunk 1 above)
Comment (codebuster22) [reply src/admin/lbps/dashboard/dashboard.html]: done.

File: src/admin/lbps/dashboard/dashboard.html (hunk 2 above)
Comment (codebuster22) [reply src/admin/lbps/dashboard/dashboard.html]: done, but need to check. For that I'll need to create a new LBP.

**Comment by codebuster22** [comment]
This is how the funding part looks like right now.
Screenshot 2021-11-08 at 9 07 38 AM

**Review by dkent600 — APPROVED** [review]

**Pull request merged**